From 727a90437e38e70a9ab8d160fa46fc022a79e577 Mon Sep 17 00:00:00 2001
From: Sean Dougherty <sdougherty@microsoft.com>
Date: Tue, 5 Nov 2024 22:41:09 +0000
Subject: [PATCH] Temporarily disabling broken test suite due to version
 mismatches between pytest-cov and python3-coverage in 3.0. Originally added
 by Francisco Huelsz Prince.

---
 meson.build | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/meson.build b/meson.build
index b29174d..1a8f0bb 100644
--- a/meson.build
+++ b/meson.build
@@ -92,11 +92,11 @@ test('legacy-tests',
      timeout: 900,
      env: test_env)
 #TODO: split out dbus tests into own test() instance, to run in parallel
-test('unit-tests',
-     pycoverage,
-     args: ['run', '-a', '-m', 'pytest', '-s', '-v', '--cov-append', meson.current_source_dir()],
-     timeout: 600,
-     env: test_env)
+# test('unit-tests',
+#      pycoverage,
+#      args: ['run', '-a', '-m', 'pytest', '-s', '-v', '--cov-append', meson.current_source_dir()],
+#      timeout: 600,
+#      env: test_env)
 
 #TODO: the coverage section should probably be cleaned up a bit
 if get_option('b_coverage')
-- 
2.45.2

